server/server.js

```diff
@@ -1,15 +1,102 @@
 const express = require("express");
 const app = express();
+const cors = require("cors");
+const dotenv = require("dotenv");
+const { Pool } = require("pg");
 const port = process.env.PORT || 5000;
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+dotenv.config();
+app.use(express.json());
+app.use(cors());
 
-// Store and retrieve your videos from here
-// If you want, you can copy "exampleresponse.json" into here to have some data to work with
-let videos = [];
+const pool = new Pool({
+  connectionString: process.env.DB_URL,
+  password: process.env.DB_PASSWORD,
+  ssl: { rejectUnauthorized: false },
+});
 
 // GET "/"
-app.get("/", (req, res) => {
-  // Delete this line after you've confirmed your server is running
-  res.send({ express: "Your Backend Service is Running" });
+app.get("/", async (req, res) => {
+  try {
+    const query = "SELECT * FROM videos";
+    const result = await pool.query(query);
+    const videos = result.rows;
+    res.status(200).json({ videos });
+  } catch (error) {
+    console.log("Error retrieving videos:", error);
+  }
+});
+
+// POST
+app.post("/", async (req, res) => {
+  try {
+    const { title, url } = req.body;
+    const rating = 0;
+    const query =
+      "INSERT INTO videos (title, url, rating) VALUES ($1,$2,$3) RETURNING *";
+    const values = [title, url, rating];
+    const result = await pool.query(query, values);
+    const newVideo = result.rows[0];
+    res.status(201).send(newVideo);
+  } catch (error) {
+    console.log("post error");
```

Comment:
We should make the server respond in this case, what error should it respond with? 🤔